concepts/functions/introduction.md

```diff
@@ -0,0 +1,119 @@
+# Introduction
+
+## Terminology
+
+You have already used and written C++ functions in a couple of concepts.
+It is time to get technical.
+The code snippet below shows the most common terms for easy reference.
+As C++ ignores whitespace, the formatting has been changed to put each element on a single line.
+
+```cpp
+// Function declaration:
+bool admin_detected(string user, string password) // Function header
```

Comment:
I missed this in my previous review: Is "Function header" an established term?

Reply:
🤯 ?!
I was so sure it is! And yet extensive search did not result in any findings. Maybe that came from some other language and I always assumed it is the same in Cpp. It seems so consequential to have a body and a header.